lib/index.js

```diff
@@ -62,7 +57,7 @@ exports.validateOperation = async ({
     url, method, headers, operation, storage,
     getInvokedCapability, documentLoader, expectedHost, expectedRootCapability,
     // TODO: support RsaSignature2018 and other suites?
-    inspectCapabilityChain, suite: [new Ed25519Signature2018()]
+    inspectCapabilityChain, suite: [new Ed25519Signature2020()]
```

Comment:
This looks like a breaking change to me, is that not the case?  If it is breaking, we just need to document as such in the changelog.

It appears to me that operations must now be signed using `Ed25519Signature2020`.

Reply:
Updated e68af7c